### ISSUE TITLE
[AOS/0.9.40] No authentication callback on a device with lower version of GPG app.

### ISSUE THREAD

**Issue by y3102sof** [playgameservices/play-games-plugin-for-unity#1876]
Please check below link.

https://github.com/playgameservices/play-games-plugin-for-unity/issues/1836
https://github.com/playgameservices/play-games-plugin-for-unity/pull/1837

That issue still occur in 0.9.40.
Please check and fix.

**Comment by MaxTyT** [comment]
Yes, I have the same. So I made a 10 second timeout. If a callback is not called I assume that there is no GPG.

**Comment by vaibhavgpt-google** [comment]
Closing this issue as part of a cleanup of older tickets. Google Play Games v1 is now deprecated. Please upgrade to version v2.1.0 or newer: https://github.com/playgameservices/play-games-plugin-for-unity/releases/tag/v2.1.0. If this issue persists on the new v2 architecture, please open a new ticket.
